Release Automation Fails to Build Helm Charts

<!-- Please answer these questions before submitting your bug report. Thanks! -->

**What version of eunomia are you using?**

eunomia version: v0.0.4

**Does this issue reproduce with the latest release?**

Yes

**What operating system and processor architecture are you using (`kubectl version`)?**

n/a

**What did you do?**

<!--
If possible, provide a recipe for reproducing the error.
A detailed sequence of steps describing what to do to observe the issue is good.
A complete runnable bash shell script is best.
-->

After creating the Eunomia v0.0.4 release the helm chart build automation that is executed by Travis CI failed to completed successfully.

**What did you expect to see?**

I expected the helm charts for the eunomia v0.0.4 release to be pushed to the gh-pages branch.

**What did you see instead?**

Here is the relevant output from the Travis CI log. The full log are be found here, https://api.travis-ci.com/v3/job/259263304/log.txt.
```
./scripts/build/publish-chart-repo.sh
+mkdir -p /tmp/helm/publish
+git -C /tmp/helm/publish clone -b gh-pages git@github.com:KohlsTechnology/eunomia.git .
Cloning into '.'...
Warning: Permanently added the RSA host key for IP address '140.82.114.4' to the list of known hosts.

+version=v0.0.4
+envsubst
+helm lint deploy/helm/eunomia-operator
2019/11/21 22:44:59 found symbolic link in path: /home/travis/gopath/src/github.com/KohlsTechnology/eunomia/deploy/helm/eunomia-operator/crds resolves to /home/travis/gopath/src/github.com/KohlsTechnology/eunomia/deploy/crds
==> Linting deploy/helm/eunomia-operator
[INFO] Chart.yaml: icon is recommended

1 chart(s) linted, no failures
++basename deploy/helm/eunomia-operator
+chart_dest=/tmp/helm/publish/eunomia-operator
+mkdir -p /tmp/helm/publish/eunomia-operator
+helm package -d /tmp/helm/publish/eunomia-operator deploy/helm/eunomia-operator
2019/11/21 22:44:59 found symbolic link in path: /home/travis/gopath/src/github.com/KohlsTechnology/eunomia/deploy/helm/eunomia-operator/crds resolves to /home/travis/gopath/src/github.com/KohlsTechnology/eunomia/deploy/crds
Successfully packaged chart and saved it to: /tmp/helm/publish/eunomia-operator/eunomia-operator-v0.0.4.tgz
Error: stat /home/travis/.helm/repository/local: no such file or directory
Makefile:70: recipe for target 'publish-chart-repo' failed
make: *** [publish-chart-repo] Error 1
Script failed with status 2

travis_fold:end:dpl.3
failed to deploy
```

Latest errors when releasing v0.0.5 ...
```
./scripts/build/publish-chart-repo.sh
+mkdir -p /tmp/helm/publish
+git -C /tmp/helm/publish clone -b gh-pages git@github.com:KohlsTechnology/eunomia.git .
Cloning into '.'...
Warning: Permanently added the RSA host key for IP address '140.82.114.3' to the list of known hosts.
+version=v0.0.5
+envsubst
+helm lint deploy/helm/eunomia-operator
2019/12/05 19:19:19 found symbolic link in path: /home/travis/gopath/src/github.com/KohlsTechnology/eunomia/deploy/helm/eunomia-operator/crds resolves to /home/travis/gopath/src/github.com/KohlsTechnology/eunomia/deploy/crds
==> Linting deploy/helm/eunomia-operator
[INFO] Chart.yaml: icon is recommended
1 chart(s) linted, no failures
++basename deploy/helm/eunomia-operator
+chart_dest=/tmp/helm/publish/eunomia-operator
+mkdir -p /tmp/helm/publish/eunomia-operator
+helm package -d /tmp/helm/publish/eunomia-operator deploy/helm/eunomia-operator
2019/12/05 19:19:20 found symbolic link in path: /home/travis/gopath/src/github.com/KohlsTechnology/eunomia/deploy/helm/eunomia-operator/crds resolves to /home/travis/gopath/src/github.com/KohlsTechnology/eunomia/deploy/crds
Successfully packaged chart and saved it to: /tmp/helm/publish/eunomia-operator/eunomia-operator-v0.0.5.tgz
Error: stat /home/travis/.helm/repository/local: no such file or directory
Makefile:68: recipe for target 'publish-chart-repo' failed
make: *** [publish-chart-repo] Error 1
Script failed with status 2
failed to deploy

--- a/scripts/build/publish-chart-repo.sh
+++ b/scripts/build/publish-chart-repo.sh
@@ -23,6 +23,7 @@ git -C "$GITHUB_PAGES_DIR" clone -b "$GITHUB_PAGES_BRANCH" "git@github.com:$GITH
 
 # build
 
+helm init --client-only
 version=${TRAVIS_TAG} envsubst < "$HELM_CHARTS_SOURCE"/Chart.yaml.tpl  > "$HELM_CHARTS_SOURCE"/Chart.yaml
 helm lint "$HELM_CHARTS_SOURCE"
 chart_dest=$HELM_CHART_DEST/"$(basename "$HELM_CHARTS_SOURCE")"

